packages/addons/addon-depends/audio-tools-depends/alsa-plugins/package.mk

```diff
@@ -16,6 +16,10 @@ if [ "$PULSEAUDIO_SUPPORT" = yes ]; then
   SUBDIR_PULSEAUDIO="pulse"
 fi
 
-PKG_CONFIGURE_OPTS_TARGET="--with-plugindir=/usr/lib/alsa"
-PKG_MAKE_OPTS_TARGET="SUBDIRS=$SUBDIR_PULSEAUDIO"
-PKG_MAKEINSTALL_OPTS_TARGET="SUBDIRS=$SUBDIR_PULSEAUDIO"
+pre_configure_target() {
+  PKG_CONFIGURE_OPTS_TARGET="--with-plugindir=/usr/lib/alsa"
+}
+
+makeinstall_target() {
+  make install DESTDIR=$INSTALL $PKG_MAKEINSTALL_OPTS_TARGET
```

Comment:
`PKG_MAKEINSTALL_OPTS_TARGET ` is undefined here. Do you need `makeinstall_target` at all, just use the `make install` in `scripts/build`?

Reply:
I use a custom `makeinstall_target` to avoid polluting `SYSROOT` with libraries which may be detected and linked in the image. I use the same line as in `scripts/build`, just in case `PKG_MAKEINSTALL_OPTS_TARGET` is set.

Reply:
You can remove the reference to `$PKG_MAKEINSTALL_OPTS_TARGET` as that's a method to pass information to `scripts/build`, which you are not using. Should the variable ever be set to a default value in future (unlikely as it's intended to be set by package.mk) then your package might be adversely affected, in which case I'd recommend removing it so that the behaviour of your package remains constant.

However if you choose to leave it then fine, it's just a potential future issue to be aware of.